dictionaries/en_US/src/en_US.txt

```diff
@@ -4095,6 +4095,8 @@ counterinsurgents
 countermaid
 counteropening
 counterphobic
+counterparties
```

Comment:
Please `counterparty` and `counterparties` to [`dictionaries/en_shared/src/shared-additional-words.txt`](https://github.com/streetsidesoftware/cspell-dicts/blob/main/dictionaries/en_shared/src/shared-additional-words.txt).